src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/AvoidOutParametersTests.cs

```diff
@@ -456,14 +456,14 @@ public class InterfaceImplicitImpl : IInterface
 }
 
 public class InterfaceExplicitImpl : IInterface
-{
+{Do not use banned APIs
```

Comment:
Changes in this file don't look correct.

Reply:
Sorry, I will redo the PR more carefully.